epub34/overview/index.html

```diff
@@ -103,15 +103,21 @@ <h3>Reading this document</h3>
 				<p id="fig-epub-structure">The following figure visually represents the structure of an EPUB
 					publication:</p>
 
-				<img src="images/epub.svg" width="600" aria-details="fig-epub-structure-diagram"
-					alt="Visual structure of the main constituents of an EPUB publication" />
+				<figure>
+					<figcaption>
+						Structure of an EPUB publication.
+					</figcaption>
+					<img src="images/epub.svg" width="600" aria-details="fig-epub-structure-diagram"
+						alt="Visual structure of the main constituents of an EPUB publication"
+						style="background-color: transparent;" />
+				</figure>
 
 				<details id="fig-epub-structure-diagram" class="desc">
 					<summary style="font-weight: normal; font-style: italic">Image description</summary>
 					<p>'EPUB (OCF) container' as the outer most component which encapsulates the 'EPUB publication'
 						containing two documents 'Package Document' and 'EPUB navigation document' along with a
 						inner component labelled 'publication resources' which contains multiple 'EPUB content
-						documents' (XHTML, SVG), and multiple 'Other resources' (CSS, png, mp3, mov, etc.). </p>
+						documents (XHTML, SVG)', and multiple 'Other resources (CSS, png, mp3, mov, etc.)'. </p>
```

Comment:
The use of quotes here suggests direct citation, but the text in the summary and in the image are slightly different. Shouldn't the image use "EPUB content documents", for example? It also has the singular "Other resource" and an ellipsis instead of "etc.". I hate to make image manipulation work for you, but having these in sync would be best.

Reply:
I decided not to touch the graphics, but went through the alt text to sync them. This is an old thing that nobody discovered so far... I believe whatever is in quote here is now the same as in the figure (except for the new line in the diagram).